src/switch_ivr_bridge.c

```diff
@@ -772,7 +772,9 @@ static void *audio_bridge_thread(switch_thread_t *thread, void *obj)
 #if DEBUG_RTP
 		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_NOTICE, "Audio bridge thread: #18 %p -> %p\n", (void*)session_a, (void*)session_b);
 #endif
+			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_NOTICE, "Parsing next event...\n");
 			switch_ivr_parse_next_event(session_a);
+			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_NOTICE, "Done parsing next event.\n");
 			msg.message_id = SWITCH_MESSAGE_INDICATE_BRIDGE;
```

Comment:
Better add a log in SWITCH_MESSAGE_INDICATE_BRIDGE and SWITCH_MESSAGE_INDICATE_UNBRIDGE in switch_core_media.c. The switch_rtp_ready is using a lock and it might also cause the issue.